### ISSUE TITLE
Har vore inne på det med avl før, meeeen

### ISSUE THREAD

**Issue by Hypsibius** [realfagstermer/realfagstermer#327]
Vi har av emneord

Naturlig utvalg, BF naturlig seleksjon (so far, so good)

Seksuell seleksjon (kan eg absolutt vere med på å skille ut)

Seleksjon (her er det hummar og kanari i treffa. Mykje evolusjonært, der det meinast  naturleg eller seksuell seleksjon). 

Vi manglar: Kunstig utvalg, eksempelvis BF kunstig seleksjon, (målretta) avl, (målretta) oppdrett etc etc. 

Eg likar ikkje å ha seleksjon (eller utvalg) ståande åleine så lenge vi snakkar evolusjon. Vi bør presse det inn i anten naturleg eller styrt utval. Kva synest dokke andre?

**Comment by bibliomari** [comment]
@BioHeidi

**Comment by BioHeidi** [comment]
Ja, enig i at vi godt kan spesifisere det i naturlig/seksuell seleksjon på de treffene vi nå får på seleksjon. Og plukke ut de som handler om kunstig utvalg. Synes termforslaget der er bra.
